FibModified.asm

```diff
@@ -0,0 +1,116 @@
+# COMP3410 Program Template
+# Author: William C. Moyer
+# Assignment: PA[X]
+# Date: Date of submission
+
+# Turn in one .asm file per assignment component
+# Remember to submit it as a pull request to the GitHub repo for the assignment
+
+# fibonacci.asm
+# Input: None
+# Output: Print to terminal N Fibonacci numbers
+
+##########################################################
+# .data segment. Constant and variable definitions go here.
+##########################################################
+
+	.data
+
+fibs:	.word   0 : 9         # create an array variable named "fibs" of 9 word-length elements (4 bytes each)
+size: .word 0 #initialize the size
+welcome: .asciiz "Please enter a number from 1 - 9:  " # displays a welcome message for user input
+error: .asciiz "YOU ENTERED AN INVALID NUMBER! RUN THE PROGRAM AGAIN." #invalid number string notification
+
+
+
+
+
+
+
+	
+	
+###############################################
+# .text segment. Assembly instructions go here.
+###############################################
+	.text
+	try: # restarts if the incorrect value is input
+	      # print out the welcome message and ask for user input
+	      li $v0, 4
+              la $a0, welcome
+              syscall
+              # get the user input
+	      li $v0, 5
+              syscall
+              sw $v0, size # save the user input into size
```

Comment:
May not be best idea to write over size, but you're checking for bad input so OK.